docs/modules/trino/pages/usage_guide/catalogs/generic.adoc

```diff
@@ -0,0 +1,55 @@
+= Generic
+
+As Trino supports a lots of different connectors, we can't possible cover all the available connectors.
+In case the trino-operator does not support a specific connector you can use this connector to pull in any connector Trino supports.
+You are one you own figuring out how to configure and use this connector and we can't possible support all the catalogs available.
+
+With this out of the way, this is how you can e.g. use the https://trino.io/docs/current/connector/postgresql.html[PostgreSQL connector]:
+
+== Example generic catalog properties
+[source,yaml]
+----
+apiVersion: trino.stackable.tech/v1alpha1
+kind: TrinoCatalog
+metadata:
+  # The name of the catalog as it will appear in Trino
+  name: postgres
+  # TrinoCluster can use these labels to select which catalogs to include
+  labels:
+    trino: simple-trino
+spec:
+  connector:
+    generic:
+      connectorName: postgresql
+      properties: # optional
```

Comment:
This is only optional for a few connectors right?

Reply:
Si, e.g. tpch, tpcds or the storage connector we use for a customer